buildenv/jenkins/jobs/builds/Build-Test-Any-Platform

```diff
@@ -32,6 +32,14 @@ timeout(time: 10, unit: 'HOURS') {
         node(SETUP_LABEL) {
             try{
                 checkout scm
+                // For OpenJ9 PRs, pull the pipeline code from the PR merge ref in order to test the pipeline changes
```

Comment:
could this be moved into a function?

Reply:
I can't move it to a function in a common file because we would continue to load common functions form master instead of the PR.

Reply:
k

Reply:
k

Reply:
There is another `checkout scm`  for the test script (L67). Is it ignored on purpose?

Reply:
That code is not used in pipeline builds and will be dead as soon as we turn off legacy PR builds. Then it can be cleaned up via #2743.
https://github.com/eclipse/openj9/pull/2743/files#diff-b19f95b3c3121dd4c7d986c2633a9deaL59